### PR TITLE
TCJA reconciliation reform file

### PR DESCRIPTION
**Pull request by codykallen**
This PR adds a reform file for the reconciliation version of the TCJA.

### CODE REVIEW

File: taxcalc/reforms/TCJA_Reconciliation.json [hunk 1]
```diff
@@ -0,0 +1,174 @@
+// Title: Tax Cuts and Jobs Act, Reconciliation verson
```
Comment (MaxGhenis) [inline taxcalc/reforms/TCJA_Reconciliation.json]: version

File: taxcalc/reforms/TCJA_Reconciliation.json [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+// Reform_Description:
+// -  New personal income tax schedule (regular/non-AMT/non-pass-through) (1)
+// -  New pass-through income tax schedule (2)
+// -  New standard deductions (3)
+// -  Repeal personal exemption (4)
+// -  Modification to child tax credit, nonrefundable dependent credits (5)
+// -  Modification of Alternative Minimum Tax exemption (6)
+// -  Repeal of certain above the line deductions (7)
+// -  Changes to itemized deductions (8)
+// -  Switch to chained CPI from CPI-U for tax paramter adjustment (9)
```
Comment (MaxGhenis) [inline taxcalc/reforms/TCJA_Reconciliation.json]: parameter

**Comment by martinholmer** [comment]
@codykallen,  We all appreciate your fast, high-quality work, but this time don't forget to run all the tests before you merge this PR.  

In #1756, you forgot to update the user documentation (which was done in #1757).

Just run all the tests **locally** before your final commit to this pull request.

File: taxcalc/reforms/TCJA_Reconciliation.json [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+            {"2018": [0.37],
+             "2026": [0.396]},
+        "_II_brk1":
+            {"2018": [[9525, 19050, 9525, 13600, 19050]],
+             "2026": [[11242, 22484, 11242, 16094, 22484]]},
+        "_II_brk2":
+            {"2018": [[38700, 77400, 38700, 51800, 77400]],
+             "2026": [[45751, 91502, 45751, 61242, 91502]]},
+        "_II_brk3":
+            {"2018": [[82500, 165000, 82500, 82500, 165000]],
```
Comment (MattHJensen) [inline taxcalc/reforms/TCJA_Reconciliation.json]: Shouldn't single be 70000?

Comment (MattHJensen) [reply taxcalc/reforms/TCJA_Reconciliation.json]: (and separate?)

Comment (codykallen) [reply taxcalc/reforms/TCJA_Reconciliation.json]: No. The description is the two-pager you sent is incorrect. The correct number, at least according to the text of the bill and the section-by-section summary, is $82,500.

File: taxcalc/reforms/TCJA_Reconciliation.json [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+             "2026": [[110791, 184571,  92286, 158169, 184571]]},
+        "_II_brk4":
+            {"2018": [[157500, 315000, 157500, 157500, 315000]],
```
Comment (MattHJensen) [inline taxcalc/reforms/TCJA_Reconciliation.json]: Single/Separate -> 160000

File: taxcalc/reforms/TCJA_Reconciliation.json (hunk 3 above)
Comment (MattHJensen) [reply taxcalc/reforms/TCJA_Reconciliation.json]: Got it. Thanks.

File: taxcalc/reforms/TCJA_Reconciliation.json (hunk 4 above)
Comment (codykallen) [reply taxcalc/reforms/TCJA_Reconciliation.json]: No, the bill specifies $157,500. See the link in the JSON, page 3 of the legislative text.

**Comment by MattHJensen** [comment]
pg 151:

> The phaseout thresholds are increased to $1,000,000 for married taxpayers filing a joint return, and $500,000 for all other taxpayers (other than estates and trusts). These amounts are indexed for inflation.

**Comment by MattHJensen** [comment]
This looks good to me other than my one outstanding [comment](https://github.com/open-source-economics/Tax-Calculator/pull/1759#issuecomment-352142260). 

@evtedeschi3, @GoFroggyRun, what do you think?

File: taxcalc/reforms/TCJA_Reconciliation.json (hunk 4 above)
Comment (MattHJensen) [reply taxcalc/reforms/TCJA_Reconciliation.json]: Got it. Thanks.

Comment (GoFroggyRun) [reply taxcalc/reforms/TCJA_Reconciliation.json]: 157,500 seems to be right according to http://docs.house.gov/billsthisweek/20171218/CRPT-115HRPT-466.pdf, page 2 and 3.

**Comment by GoFroggyRun** [comment]
I've checked all 2026 values, and they look good to me. +1

**Comment by evtedeschi3** [comment]
Looks good to me!

**Comment by MattHJensen** [comment]
@codykallen, are [this](https://github.com/open-source-economics/Tax-Calculator/pull/1759#issuecomment-352142260) and [this](https://github.com/open-source-economics/Tax-Calculator/pull/1759#issuecomment-352137774) good?

**Comment by GoFroggyRun** [comment]
Seems like [500000, 1000000, 500000, 500000, 1000000] are the right values for '_AMT_em_ps' in 2018.

**Comment by codykallen** [comment]
@martinholmer, running `docs/make_index.py` does not appear to have changed any files.

**Comment by codykallen** [comment]
@MattHJensen, trying to update `docs/index.html` doesn't seem to be doing anything. Do you have a specific recommendation?

**Comment by MattHJensen** [comment]
> @MattHJensen, trying to update docs/index.html doesn't seem to be doing anything. Do you have a specific recommendation?

I believe that means `docs/index.html` doesn't need to be updated. I am going to merge this so that @martinholmer can respond to your question on a more leisurely timeline. 

Thanks very much to @codykallen for contributing this and to @evtedeschi3 and @GoFroggyRun for your review. I expect that some of our users will be working with this tonight, and I will follow up if they have any feedback.

**Comment by martinholmer** [comment]
@codykallen,  Adding only a reform file did not trigger a change in user documentation, but adding the new reform file did require a change in the `taxcalc/reforms/REFORMS.md` file (which has been taken care of in PR#1760).  Thanks for all the work involved in quickly developing the JSON reform file for the conference committee bill.

**Pull request merged**